packages/airnode-protocol/package.json

```diff
@@ -44,7 +44,7 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "4.4.2",
-    "ethers": "^5.4.5"
+    "@openzeppelin/contracts": "4.7.0",
```

Comment:
@bbenligiray you were against upgrading this because the ABI could change right?

Reply:
It will change the contract bytecode and contract deployment validations will start failing
https://github.com/api3dao/airnode/blob/master/packages/airnode-protocol/package.json#L21-L22

Reply:
Why are we not running that on CI?

Reply:
Created https://github.com/api3dao/airnode/issues/1305